app/src/main/kotlin/cz/covid19cz/erouska/utils/Analytics.kt

```diff
@@ -8,6 +8,17 @@ object Analytics {
     const val KEY_EXPORT_DOWNLOAD_STARTED = "key_export_download_started"
     const val KEY_EXPORT_DOWNLOAD_FINISHED = "key_export_download_finished"
 
+    const val KEY_SHARE_APP = "click_share_app"
```

Comment:
I also have a preference to "tap". OnClickListener is more like a relict from distant past when Android was being born :) But no blocker.